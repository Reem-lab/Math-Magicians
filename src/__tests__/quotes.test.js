import renderer from 'react-test-renderer';
import Quotes from '../component/quotes';

it('renders correctly', () => {
  const tree = renderer
    .create(<Quotes />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
