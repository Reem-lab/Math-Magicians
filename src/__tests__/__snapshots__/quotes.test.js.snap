// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="quotes"
>
  <p>
    Mathmatics is not about numbers, equations, Computations or Algorithms; it is about understanding -- William paul Thurston
  </p>
</div>
`;
